implementations/micrometer-registry-dynatrace/src/main/java/io/micrometer/dynatrace/v2/DynatraceExporterV2.java

```diff
@@ -174,6 +174,10 @@ private String createCounterLine(Meter meter, Measurement measurement) {
 
     private Stream<String> toSummaryLine(Meter meter, HistogramSnapshot histogramSnapshot, TimeUnit timeUnit) {
         long count = histogramSnapshot.count();
+        if (count < 1) {
```

Comment:
Is there a difference on the backend between sending a zero measurement and not sending a measurement at all?
E.g.: If the service is down (or the meter got removed from Micrometer) vs. it is up but it is not receiving any traffic.

Reply:
As far as I am aware, there currently is no difference for summary metrics.